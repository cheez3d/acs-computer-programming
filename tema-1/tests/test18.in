mama tata sora frate
628#88777720886622244#4440663376668066337666282
tatal meu este fiul bunicii
2
bunicii mamei
tatal fratele
sora fiu nora

