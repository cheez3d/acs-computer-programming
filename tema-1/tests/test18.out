62620828207777666777203337772833
matusa unchi nepot nepoata
3337772833555330633880337777833033344488555062633444
3
77776667772 33344488 666667772 
77776667772 1
0
880666 777766 677720 667772 333444 
